modules/installation-osp-setting-cloud-provider-options.adoc

```diff
@@ -0,0 +1,65 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-user.adoc
+
+[id="installation-osp-setting-cloud-provider-options_{context}"]
+= Setting cloud provider options
+
+Optionally, you can edit the cloud provider configuration for your cluster. The cloud provider configuration controls how {product-title} interacts with {rh-openstack-first}.
+
+For a complete list of cloud provider configuration parameters, see link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
+
+.Prerequisites
+
+* Have manifest files generated for your cluster.
+** If you don't have manifest files, generate them by running `openshift-install --dir <destination_directory> create manifests` from a command line.
+
+.Procedure
+
+. In a text editor, open the cloud-provider configuration manifest file. For example:
++
+[source,terminal]
+----
+$ vi openshift/manifests/cloud-provider-config.yaml
+----
+
+. Modify the options based on link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
++
+Configuring Octavia for load balancing is a common case. For example:
++
+[source,yaml]
+----
+#...
+[LoadBalancer]
+use-octavia=true <1>
+lb-provider = "amphora" <2>
+floating-network-id="d3deb660-4190-40a3-91f1-37326fe6ec4a"<3>
+floating-subnet-id="4a2fe995-84d4-4eac-8bcf-c5624f63dba0"
+subnet-id="e3d26bd3-0b6b-4c4d-b184-71c5be59b8b2"
```

Comment:
Hm, we should probably either explain this or just omit this option (cloud-provider will autodetect it). Basically it's about in which subnet to create the load balancers. We've never tested anything other than the subnet used by the nodes, so I'd rather remove this line.

@iamemilio: I just realized that we haven't checked how the autodetection works if you have nodes in another subnet. There may be dragons.

Reply:
@eurijon No qualms with removing this line? 

```suggestion
```

Reply:
It can be removed as it's autodetected by the cloud-provider

Reply:
TY.